docs/docs/monitoring/analytics.mdx

```diff
@@ -76,11 +76,25 @@ will be disabled.
 
 TODO: general instructions.
```

Comment:
I wasn't sure what was intended here.

Reply:
is there a need for a user to be created for Analytics to connect to PG? which permissions?

Reply:
I've added more details on auth options in this section: https://github.com/RasaHQ/rasa/pull/11653/commits/c6a61db250526df11427351c0049ac655e1bbfa6